lib/paper_trail.rb

```diff
@@ -26,6 +26,8 @@ module PaperTrail
     named created_at.
   EOS
 
+  RAILS_GTE_7_0 = ::ActiveRecord.gem_version >= ::Gem::Version.new("7.0.0")
```

Comment:
Nice use of `Gem::Version`

Reply:
Thanks. Just trying to stick to the gem's dialect - got it from `ModelConfig`.